Solutions/Windows Security Events/Hunting Queries/Suspicious_enumeration_using_adfind.yaml

```diff
@@ -29,6 +29,7 @@ tags:
   - Solorigate
   - NOBELIUM
 query: |
+  // Adjust lookupwindows for aggregate interval 
   let lookupwindow = 2m;
   let threshold = 3; //number of commandlines in the set below
   let DCADFSServersList = dynamic (["DCServer01", "DCServer02", "ADFSServer01"]); // Enter a reference list of hostnames for your DC/ADFS servers
```

Comment:
Now we have some device info in Sentinel we could replace this static list with  a dynamic lookup of that data to determine DCs.

Reply:
Thanks for the idea @petebryan. I spent an hour to find out a concrete way to differentiate domain controllers from other devices using deviceinfo data, but I couldn't find any definitive way. I'll further research on this. In the meantime if you have any example rule/query please let me know.